src/main/java/com/gmail/axelwerst/Shape.java

```diff
@@ -0,0 +1,76 @@
+package com.gmail.axelwerst;
+
+    abstract class Shape {
+        public abstract double calculateArea();
+        public abstract double calculatePerimeter();
+
+        public abstract String displayInfo();
```

Comment:
Method `displayInfo` can be `void` as it's only responsibility to display the information to terminal and we don't actually use the returned result anywhere.